package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "test": "atom --test ./spec",
     "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
```

Comment:
This isn't needed. You can just run `npm run lint -- --fix`.

Reply:
Yes. 😁 I can revert it.